crates/hyperqueue/src/client/output/cli.rs

```diff
@@ -888,9 +893,32 @@ pub fn format_task_duration(
             start_date,
             end_date,
             ..
-        } => *end_date - *start_date,
+        } => (Option::Some(*start_date), Option::Some(*end_date))
+    }
+}
+
+pub fn format_task_time(
```

Comment:
You can replace this with something like `duration.map(|v| v.to_string()).unwrap_or_else(|| "".to_string())`.